app/renderer/components/preferences/payment/ledgerRecovery.js

```diff
@@ -31,8 +31,10 @@ class LedgerRecoveryContent extends ImmutableComponent {
     this.props.handleRecoveryKeyChange(e.target.value)
   }
 
-  clearRecoveryStatus () {
-    this.props.hideAdvancedOverlays()
+  clearRecoveryStatus (success) {
```

Comment:
i think the changes to this and the following files aren't supposed to be here

Reply:
I went back to far on the squash

Reply:
@diracdeltas Is this back to normal with the right changes?